### DISCUSSION TITLE
Is there any way to compile cefsharp/120 that can be used in. net4.0+win10_x86？？？

### DISCUSSION THREAD

**Discussion by jozzlee** [cefsharp/CefSharp#4751] [Q&A]
I just took over a project developed by someone else **12 years** ago. 
It does not support ES11 and relies on several DLLs without source code. Can only run on **.net4.0 x86**. 
The good news is that it is **available in win10**.

**Comment by amaitland** [comment]
The minimum support version is `.Net 4.6.2`.

Building targeting older versions is not supported.
